### PR TITLE
Start glance plugin - GET images call

### PR DESCRIPTION
**Pull request by tawalton**

### CODE REVIEW

File: mimic/rest/glance_api.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    app = MimicApp()
+
+    @app.route('/v2/<string:tenant_id>/images', methods=['GET'])
+    def get_images(self, request, tenant_id):
+        """
+        Returns a list of glance images. Currently there is no provision
+        for shared versus unshared images in the response
+        """
+        request.setResponseCode(200)
+        return json.dumps(get_images())
```
Comment (lekhajee) [inline mimic/rest/glance_api.py]: shouldn't this return a list ?

Comment (tawalton) [inline mimic/rest/glance_api.py]: Yes, it should, but after talking with @glyph we decided to start with one image for the sake of landing this PR. We will need to add additional images in the future.  There is an earlier PR that had all the current images included: https://github.com/rackerlabs/mimic/blob/plugins_for_reach/mimic/canned_responses/json/glance/glance_images_json.py

Comment (lekhajee) [inline mimic/rest/glance_api.py]: right, even with the one image, I would think it would exist with the `images` list object.

Comment (tawalton) [inline mimic/rest/glance_api.py]: I am not following what you are saying. Can you provide an example or we can discuss off line?

Comment (glyph) [inline mimic/rest/glance_api.py]: I *think* what Lekha is saying refers to the structure of the data which `get_images()` returns, but this line of code is a bit removed from the problem, so I'll put the comment there :)

File: mimic/canned_responses/images_json.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+"""
+Glance images
+"""
+
+images = {
+    "images":
+        {
```
Comment (glyph) [inline mimic/canned_responses/images_json.py]: This should be a list; i.e.

`"images": {...}`

should be

`"images": [{...}]`.

**Comment by lekhajee** [comment]
+1

**Comment by lekhajee** [comment]
@tawalton : Needs a rebase to be merged.

**Comment by glyph** [comment]
In the future, I'd appreciate it if we could merge master rather than rebasing pushed branches.

**Pull request merged**